src/averageAges.js

```diff
@@ -55,7 +64,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const childrenWithMotherInList = people
+    .filter(child => people
+      .some(person => person.name === child.mother));
```

Comment:
Very informative name for variable, but please try to make it shorter 🥲

Reply:
And again, filter array once

Reply:
> Very informative name for variable, but please try to make it shorter 🥲

As you wish!